esp-hal/src/lib.rs

```diff
@@ -506,8 +506,13 @@ use crate::{
 pub struct Config {
     /// The CPU clock configuration.
     pub cpu_clock: CpuClock,
+
     /// Enable watchdog timer(s).
     pub watchdog: WatchdogConfig,
+
+    /// PSRAM configuration.
+    #[cfg(any(feature = "quad-psram", feature = "octal-psram"))]
+    pub psram: psram::PsramConfig,
```

Comment:
Is it worth making this `Option`? idk if psram power consumption is significant enough to warrant it

Reply:
Alas, that makes no sense. The PSRAM shares a bunch of lines with the flash, so if they can be powered separately, even if the PSRAM could be powered down, the communication lines would keep it powered - through ESD diodes, which is definitely NOT what anyone wants.